### PR TITLE
Top level tabs Fixes #293

### LINKED ISSUE #293: Add top level navigation tabs

### PR DESCRIPTION
**Pull request by pmuir**
Ready for review.

### CODE REVIEW

File: src/app/app.module.ts [hunk 1]
```diff
@@ -35,11 +35,19 @@ import { HeaderComponent } from './header/header.component';
...
 import { WorkItemListModule }             from './work-item/work-item-list/work-item-list.module';
 
+// Main areas
+import { HomeComponent } from './home/home.component';
+import { CodeComponent } from './code/code.component';
+import { TestComponent } from './test/test.component';
+import { PipelineComponent } from './pipeline/pipeline.component';
+import { HypothesisComponent } from './hypothesis/hypothesis.component';
+import { SettingsComponent } from './settings/settings.component';
+
```
Comment (sanbornsen) [inline src/app/app.module.ts]: Instead of component, each of these main component will be a Module.

**Comment by joshuawilson** [comment]
[test]

**Comment by joshuawilson** [comment]
[test]

File: package.json [hunk 2]
```diff
@@ -65,6 +65,7 @@
     "angular-in-memory-web-api": "0.1.13",
     "bootstrap": "^3.3.7",
     "core-js": "^2.4.1",
+    "ng2-bootstrap": "^1.1.16",
```
Comment (sanbornsen) [inline package.json]: Why do we need this?

**Comment by joshuawilson** [comment]
[test]

**Pull request merged**